ci-operator/step-registry/ipi/conf/vsphere/template/ipi-conf-vsphere-template-commands.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+
+echo "$(date -u --rfc-3339=seconds) - sourcing context from vsphere_context.sh..."
+# shellcheck source=/dev/null
+declare vsphere_datacenter
+declare vsphere_datastore
+declare vsphere_url
+declare vsphere_cluster
+source "${SHARED_DIR}/vsphere_context.sh"
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/govc.sh"
+export HOME="${HOME:-/tmp/home}"
+export XDG_RUNTIME_DIR="${HOME}/run"
+export REGISTRY_AUTH_PREFERENCE=podman # TODO: remove later, used for migrating oc from docker to podman
+mkdir -p "${XDG_RUNTIME_DIR}"
+
+function getVersion() {
+  local release_image=""
+  if [ -n "${RELEASE_IMAGE_INITIAL-}" ]; then
+    release_image=${RELEASE_IMAGE_INITIAL}
+  elif [ -n "${RELEASE_IMAGE_LATEST-}" ]; then
+    release_image=${RELEASE_IMAGE_LATEST}     
+  fi
+  
+  local version=""
+  if [ ${release_image} != "" ]; then
+    oc registry login
+    version=$(oc adm release info ${release_image} --output=json | jq -r '.metadata.version' | cut -d. -f 1,2)    
+  fi
+  echo "${version}"
+}
+
+VERSION=$(getVersion)
```

Comment:
Let us add one more line for debugging:
```
echo "${VERSION}"
```

Reply:
seems the function didn't got a integer here. where we can check the value of RELEASE_IMAGE_INITIAL- and RELEASE_IMAGE_LATEST- ?

Reply:
From https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/41996/rehearse-41996-periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-vsphere-ipi-template-f14/1690912330663071744/artifacts/vsphere-ipi-template-f14/ipi-conf-vsphere-template/build-log.txt:
```
4.14 does not support template setting
```
The version can be printed out, so I suspect there are some other message from `getVersion`  beside version info.

Reply:
Interesting, add one more line for debugging:
```
set -x
```

Reply:
Okay, the whole output of is `getVersion`:
```
Saved credentials for registry.apps.build01-us-west-2.vmc.ci.openshift.org into /alabama/run/containers/auth.json
4.14
```

Reply:
We can move the `oc registry login` command out of `getVersion` function to line 20.